src/components/AdmissionForm/terms_conditions.tsx

```diff
@@ -0,0 +1,60 @@
+const TandC = () => {
```

Comment:
Do not use short names as that might lead to ambiguity.
```suggestion
const TermsAndConditions = () => {
```